README.md

```diff
@@ -19,8 +19,8 @@ There are also 3 subcommands to target specific scenarios:
 #### Common options for all commands
 - `--no-restore`: Doesn't execute an implicit restore before formatting.
 - `--include-generated`: Format files generated by the SDK
-- `--include`:  A list of relative file or folder paths to include in formatting. All files are formatted if empty.
-- `--exclude`: A list of relative file or folder paths to exclude from formatting.
+- `--include`:  A space-separated list of relative file or folder paths to include in formatting. All files are formatted if empty. (e.g. `--include **/*.abc **/*.xyz`)
+- `--exclude`: A space-separated list of relative file or folder paths to exclude from formatting. (e.g. `--exclude **/*.abc **/*.xyz`)
```

Comment:
```suggestion
- `--include`:  A space-separated list of relative file or folder paths to include in formatting. All files are formatted if empty. (e.g. `--include path/to/file.cs path/to/folder folder2/*.vb`)
- `--exclude`: A space-separated list of relative file or folder paths to exclude from formatting. (e.g. `--exclude path/to/file.cs path/to/folder folder2/*.vb` )
```

It feels like using the "real world" file extensions in the inline examples will help make things clear.

Reply:
Agreed, thanks!